app/src/operations/mail/index.ts

```diff
@@ -14,45 +14,76 @@ export default defineOperationApp({
 			label: '$t:operations.mail.to',
 			text: Array.isArray(to) ? to.join(', ') : to,
 		},
+		{
+			label: '$t:type',
+			text: type,
+		},
 		{
 			label: '$t:operations.mail.body',
 			text: body,
 		},
 	],
-	options: [
-		{
-			field: 'to',
-			name: '$t:operations.mail.to',
-			type: 'csv',
-			meta: {
-				width: 'full',
-				interface: 'tags',
-				options: {
-					placeholder: '$t:operations.mail.to_placeholder',
-					iconRight: 'alternate_email',
+	options: (panel) => {
+		return [
+			{
+				field: 'to',
+				name: '$t:operations.mail.to',
+				type: 'csv',
+				meta: {
+					width: 'full',
+					interface: 'tags',
+					options: {
+						placeholder: '$t:operations.mail.to_placeholder',
+						iconRight: 'alternate_email',
+					},
 				},
 			},
-		},
-		{
-			field: 'subject',
-			name: '$t:subject',
-			type: 'string',
-			meta: {
-				width: 'full',
-				interface: 'input',
-				options: {
-					iconRight: 'title',
+			{
+				field: 'subject',
+				name: '$t:subject',
+				type: 'string',
+				meta: {
+					width: 'full',
+					interface: 'input',
+					options: {
+						iconRight: 'title',
+					},
 				},
 			},
-		},
-		{
-			field: 'body',
-			name: '$t:operations.mail.body',
-			type: 'text',
-			meta: {
-				width: 'full',
-				interface: 'input-rich-text-md',
+			{
+				field: 'type',
+				name: '$t:type',
+				type: 'string',
+				schema: {
+					default_value: 'input-rich-text-md',
+				},
+				meta: {
+					interface: 'select-dropdown',
+					width: 'full',
+					required: true,
```

Comment:
Let's remove required since there's default value in place, and the interface + API would still fallback to markdown if somehow a user intentionally set it to null or another value via raw value editor. Plus there aren't any required fields in other operations AFAIK haha.